README.md

```diff
@@ -2,54 +2,88 @@
 
-Pomona was born out of frustrations with the difficulties of exposing a complex business domain model
-as a RESTful web service.
-
-## Build status
-||Windows (AppVeyor)|Linux (Travis)|
+||Windows (AppVeyor)|Linux (Travis)|Version
 |:--:|:--:|:--:|
-|**develop**|[![Build Status](https://ci.appveyor.com/api/projects/status/vj3cw49n499u6046/branch/develop?svg=true)](https://ci.appveyor.com/project/Pomona/pomona/branch/develop)|[![Build Status](https://travis-ci.org/Pomona/Pomona.svg?branch=develop)](https://travis-ci.org/Pomona/Pomona)|
-|**master**|[![Build Status](https://ci.appveyor.com/api/projects/status/vj3cw49n499u6046/branch/master?svg=true)](https://ci.appveyor.com/project/Pomona/pomona/branch/master)|[![Build Status](https://travis-ci.org/Pomona/Pomona.svg?branch=master)](https://travis-ci.org/Pomona/Pomona)|
-|**future**|[![Build Status](https://ci.appveyor.com/api/projects/status/vj3cw49n499u6046/branch/future?svg=true)](https://ci.appveyor.com/project/Pomona/pomona/branch/future)|[![Build Status](https://travis-ci.org/Pomona/Pomona.svg?branch=future)](https://travis-ci.org/Pomona/Pomona)|
-
-
-## Highlights
-
-* Customizable conventions
-* Fluent style overriding for exceptions to rules
-* Autogenerated .NET client
-* Advanced query filtering
-* Simple LINQ provider
+|**develop**|[![Build Status](https://ci.appveyor.com/api/projects/status/vj3cw49n499u6046/branch/develop?svg=true)](https://ci.appveyor.com/project/Pomona/pomona/branch/develop)|[![Build Status](https://travis-ci.org/Pomona/Pomona.svg?branch=develop)](https://travis-ci.org/Pomona/Pomona)|[![NuGet](https://img.shields.io/nuget/vpre/Pomona.svg)](https://www.nuget.org/packages/Pomona)|
+|**master**|[![Build Status](https://ci.appveyor.com/api/projects/status/vj3cw49n499u6046/branch/master?svg=true)](https://ci.appveyor.com/project/Pomona/pomona/branch/master)|[![Build Status](https://travis-ci.org/Pomona/Pomona.svg?branch=master)](https://travis-ci.org/Pomona/Pomona)|[![NuGet Prerelease](https://img.shields.io/nuget/v/Pomona.svg)](https://www.nuget.org/packages/Pomona)
+|**future**|[![Build Status](https://ci.appveyor.com/api/projects/status/vj3cw49n499u6046/branch/future?svg=true)](https://ci.appveyor.com/project/Pomona/pomona/branch/future)|[![Build Status](https://travis-ci.org/Pomona/Pomona.svg?branch=future)](https://travis-ci.org/Pomona/Pomona)|[![NuGet Prerelease](https://img.shields.io/nuget/vpre/Pomona.svg?branch=future)](https://www.nuget.org/packages/Pomona)
+
+## Table of Contents
+
+1. [Overview](https://github.com/Pomona/Pomona#overview)
+2. [Documentation](https://github.com/Pomona/Pomona#documentation)
+3. [Contributing](https://github.com/Pomona/Pomona#contributing)
+4. [Get in touch](https://github.com/Pomona/Pomona#get-in-touch)
+5. [License](https://github.com/Pomona/Pomona#license)
+5. [Acknowledgements](https://github.com/Pomona/Pomona#acknowledgements)
+
+## Overview
+
+To illustrate what Pomona does, here's a diagram:
+
```

Comment:
I like the diagram but it really only captures one of the things Pomona does.
I think this might belong in the documentation instead?